opencensus/exporters/stats/stackdriver/internal/stackdriver_exporter.cc

```diff
@@ -41,6 +42,14 @@ constexpr char kProjectIdPrefix[] = "projects/";
 // Stackdriver limits a single CreateTimeSeries request to 200 series.
 constexpr int kTimeSeriesBatchSize = 200;
 
+gpr_timespec ConvertToTimespec(absl::Time time) {
```

Comment:
Consider to use absl::ToChronoTime instead of writing your own.

Reply:
Done, thx!